cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,23 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_connection = None
+        while not db_connection:
+            try:
+                db_connection = connections["default"]
+                db_connection.ensure_connection()
+                self.stdout.write("Connecting to database.")
```

Comment:
This message might be misleading as it is printed before the connection is actually confirmed. Consider moving this line after the successful connection confirmation in the else block.